bin/git/update-hooks

```diff
@@ -32,14 +73,22 @@ check_update() {
     local dest=$2
     if [ ! -e "$dest" ] ; then
         step_name "Creating new file at '$dest':"
-        git diff --color --no-index /dev/null $src
-        cp $src $dest
+        if [ $verbose == 1 ]; then
```

Comment:
No that's fine if you feel it's ok. Another route would be to pass in whether it is a new file or not and switch accordingly. Again probably more work discussing this than actually doing it. :)